subsys/bootloader/image/CMakeLists.txt

```diff
@@ -58,14 +58,19 @@ if (DEFINED CONFIG_SOC_NRF9160)
   # Override PM_PROVISION_ADDRESS since Partition Manager cannot yet place
   # partitions in OTP.
   set(otp_start_addr "0xff8108")
+  set(otp_size "768") # 192 words
   set(provision_addr ${otp_start_addr})
+  set(provision_max_size ${otp_size})
 elseif (DEFINED CONFIG_SOC_NRF5340_CPUAPP)
   # Override PM_PROVISION_ADDRESS since Partition Manager cannot yet place
   # partitions in OTP.
   set(otp_start_addr "0xff8100")
+  set(otp_size "760") # 190 words
```

Comment:
Please point to chapter in spec where this is stated

Reply:
Done